package/skelcd-control-openSUSE.spec

```diff
@@ -27,7 +27,7 @@
 #
 ######################################################################
 Name:           skelcd-control-openSUSE
-Version:        42.3.99.9
+Version:        42.3.99.10
```

Comment:
no, it have 99 as "newer" then 42.3. When master will be splitted for Leap 45, it will be renumbered to 45.0.99.X